### DISCUSSION TITLE
Unistallation

### DISCUSSION THREAD

**Discussion by archdvx** [ScoopInstaller/Scoop#4878] [Help and Questions] [answered]
Hi,
some way to run .bat file during uninstallation of program?
I try this:
"uninstaller": [
        "Invoke-ExternalCommand \"$dir\\remove_config.bat\""
    ],

remove_config.bat cleans registry.

**Comment by rashil2000** [comment, marked as answer]
You can just directly run the batch script:
```json
"uninstaller": {
    "script": "\"$dir\\remove_config.bat\""
}
```
